safe-global/safe-wallet-web
# fix: web3-onboard support for EIP6963

## What it solves
The previous version of web3-onboard we were using had a bug with EIP6963 and wallets supporting the standard were doing the wrong thing and overriding Metamask.

Resolves #

## How this PR fixes it

## How to test it
1. Next to metamask, install other wallets such as Rabby, coinbase, OKX etc
2. Web3-onboard should now display all those wallets as options to connect to
3. Clicking on metamask should open MM, clicking on Rabby should open Rabby etc

Previously the wallets were overriding metamask and it was disappearing from the list. 

## Screenshots
<img width="828" alt="grafik" src="https://github.com/safe-global/safe-wallet-web/assets/693770/be5aa7d7-90d5-4300-86fb-a9a749cc55ba">


## Checklist
